dapp/src/constants/merkleProofedAccountsToBeCompensated.json

```diff
@@ -0,0 +1 @@
43c6650becb27211afb33efd","0xa34a289c9a165c72cb5256e5fb10e9633386099cafbe4a2f4ddd6015bc31bdd2"]},"0x4B52DBd3F7643796f2e5e9d1257880f28263850f":{"address":"0x4B52DBd3F7643796f2e5e9d1257880f28263850f","eligible_ousd_value_human":"1.02","ousd_compensation_human":"1.02","ogn_compensation_w_interest_human":"0.00","ogn_compensation_human":"0.00","eligible_ousd_value":"1020264840410962304","ousd_compensation":"1020264840410962304","ogn_compensation":"0","index":228,"type":1,"duration":31104000,"rate":"250000000000000000","proof":["0x897e226595bc7b9b892bc9fb6d1240276c7737cb411f6dcf5b600381c00a60cc","0xbce6f776cc7a41b9100171d86a5b85c231142b91b985f07ad2b0fdfd83ac7735","0x447e7222bce885226cf74e6f3e185b80d8f730e3cd6acd9dab9e646e5937624f","0xd7a27ea00287607927bc97141d88f9b80210cb09545ec224376fd55a422c0a7d","0x9a383c897e5b6f003aa5de52f29edd6ae6255e58d996ad7595459506ba24181d","0x4ffbad7686561ffb6fcfcd280787343b986caa7504d22b19711fcedcb3c20b89","0x1c20c78f097ea47e60cbb82fe1c33aa66bf9e90d94429f34f840f457997519c9","0xf108a0fa3b5c75f3a81c550bbfb82a0b3c2295e1e433d4f2d7713288e1df247b","0x55e59a7bce7fb3e0bb34ddb4a13ee041c9ac355843c6650becb27211afb33efd","0xa34a289c9a165c72cb5256e5fb10e9633386099cafbe4a2f4ddd6015bc31bdd2"]},"0x4b5b754032d442831F32643f04cD6e4571865189":{"address":"0x4b5b754032d442831F32643f04cD6e4571865189","eligible_ousd_value_human":"49.12","ousd_compensation_human":"49.12","ogn_compensation_w_interest_human":"0.00","ogn_compensation_human":"0.00","eligible_ousd_value":"49123507289947856896","ousd_compensation":"49123507289947856896","ogn_compensation":"0","index":229,"type":1,"duration":31104000,"rate":"250000000000000000","proof":["0x03816ff9cdbb2cad703929867b16470b9c323ff8dca180ca97cdc2e9a5508de0","0xbce6f776cc7a41b9100171d86a5b85c231142b91b985f07ad2b0fdfd83ac7735","0x447e7222bce885226cf74e6f3e185b80d8f730e3cd6acd9dab9e646e5937624f","0xd7a27ea00287607927bc97141d88f9b80210cb09545ec224376fd55a422c0a7d","0x9a383c897e5b6f003aa5de52f29edd6ae6255e58d996ad7595459506ba24181d","0x4ffbad7686561ffb6fcfcd280787343b986caa7504d22b19711fcedcb3c20b89","0x1c20c78f097ea47e60cbb82fe1c33aa66bf9e90d94429f34f840f457997519c9","0xf108a0fa3b5c75f3a81c550bbfb82a0b3c2295e1e433d4f2d7713288e1df247b","0x55e59a7bce7fb3e0bb34ddb4a13ee041c9ac355843c6650becb27211afb33efd","0xa34a289c9a165c72cb5256e5fb10e9633386099cafbe4a2f4ddd6015bc31bdd2"]},"0x4bB0E9F8d5D95bC47D80A4E49fe33A1Dd2651507":{"address":"0x4bB0E9F8d5D95bC47D80A4E49fe33A1Dd2651507","eligible_ousd_value_human":"1.01","ousd_compensation_human":"1.01","ogn_compensation_w_interest_human":"0.00","ogn_compensation_human":"0.00","eligible_ousd_value":"1008222912088142336","ousd_compensation":"1008222912088142336","ogn_compensation":"0","index":230,"type":1,"duration":31104000,"rate":"250000000000000000","proof":["0x9fbf93143b354a13ea4e92583f969b4a1f586a95bd8e1582ecdb643e77295bf8","0xd71fb3c6aab816494d11cdcfb81e3a259ee4613034aaaee33f4c28fe317bca4a","0x447e7222bce885226cf74e6f3e185b80d8f730e3cd6acd9dab9e646e5937624f","0xd7a27ea00287
```

Comment:
Per my earlier comment, this needs to be re-generated with duration of 365 days

Reply:
👍